lib/Notes/ReferredRecord/ReferredRecord.css

```diff
@@ -5,4 +5,3 @@
   font-weight: bold;
   color: var(--color-text);
 }
-
```

Comment:
What for this change? Please check Jenkins comment on this

Reply:
This change is to fix that comment

Reply:
@Godlevskyi, there were 2 empty lines